### PR TITLE
Added re-crawl immediately and refactored JS code

### PR DESCRIPTION
**Pull request by IevgenPr**
Optimized JS code, fixed tool to work with 7.2 onwards versions, added re-crawl immediately button, added link to most recent documentation.

### CODE REVIEW

File: remove-or-recrawl-urls.html [hunk 1]
```diff
@@ -24,44 +24,35 @@
...
-  function buildFeed(header, records, footer) {
+  function buildFeed(header, records, footer, attributes ) {
     var records_xml = '';
     var rl = records.length;
+    attributes = typeof attributes !== 'undefined' ?  attributes : ''; 
     for (r = 0; r < rl; r++) {
-      if (records[r].length != 0) {
-        records_xml += '    <record url="' + records[r] + '" />\n';
-      }
+        if (records[r].length != 0) {
```
Comment (macpd) [inline remove-or-recrawl-urls.html]: why change the indentation?

**Comment by macpd** [comment]
What testing have you done for this change?

File: remove-or-recrawl-urls.html (hunk 1 above)
Comment (IevgenPr) [inline remove-or-recrawl-urls.html]: There was different indentation in different parts of the JS script. I aligned it to be the same across the script.

**Comment by IevgenPr** [comment]
To test the changes, I pushed sample URL using "Delete", "Re-Crawl" and "Re-crawl now" buttons. Verified the feeds were generated correctly, accepted by the GSA and respective action was performed by the GSA.

File: remove-or-recrawl-urls.html (hunk 1 above)
Comment (macpd) [inline remove-or-recrawl-urls.html]: great! thanks

**Pull request merged**